contracts/voting-verifier/src/execute.rs

```diff
@@ -1,6 +1,5 @@
 use cosmwasm_std::{
-    to_json_binary, Deps, DepsMut, Env, MessageInfo, QueryRequest, Response, Storage, WasmMsg,
-    WasmQuery,
+    to_binary, Deps, DepsMut, Env, MessageInfo, QueryRequest, Response, Storage, WasmMsg, WasmQuery,
```

Comment:
Why change it to use the deprecated instead?

Reply:
unintentional